geometry_msgs/Pose expected_object_pose
---